website/docs/d/application.html.markdown

```diff
@@ -66,3 +67,23 @@ output "azure_ad_object_id" {
 * `id` - The unique identifier for one of the `OAuth2Permission` or `AppRole` instances that the resource application exposes. 
 
 * `type` - Specifies whether the id property references an `OAuth2Permission` or an `AppRole`.
+
+---
+
+`oauth2_permission` block exports the following:
```

Comment:
Is this block always exported or only when it is of type `OAuth2Permission`?

Reply:
Yes it is always exported, but it might be an empty list.